datadog_checks_base/datadog_checks/base/utils/db/sql.py

```diff
@@ -13,17 +15,19 @@
 
 
 def compute_sql_signature(normalized_query):
+    # type: (str) -> Optional[str]
     """
     Given an already obfuscated & normalized SQL query, generate its 64-bit hex signature.
     """
     if not normalized_query:
         return None
     # Note: please be cautious when changing this function as some features rely on this
     # hash matching the APM resource hash generated on our backend.
-    return format(mmh3.hash64(normalized_query, signed=False)[0], 'x')
+    return format(mmh3.hash64(normalized_query, signed=False)[0], str('x'))
 
 
 def normalize_query_tag(query):
+    # type: (Union[str, unicode]) -> Union[str, unicode]
```

Comment:
Does this work?

```suggestion
    # type: (str) -> str
```

Mypy automatically does its best to deal with the str/unicode mess on Py2: 

https://mypy.readthedocs.io/en/stable/python2.html

> In Python 2 mode str is implicitly promoted to unicode, similar to how int is compatible with float. This is unlike bytes and str in Python 3, which are incompatible. bytes in Python 2 is equivalent to str. (This might change in the future.)